lib/diagnostic.rb

```diff
@@ -45,7 +55,11 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+  def solve_problems
+    return "think think think"
```

Comment:
Nice! remember that we don't need the `return` keyword in Ruby
Ruby methods always try to return something, so `"think think think"` will do